packages/ketcher-core/src/application/render/renderers/RenderersManager.ts

```diff
@@ -43,6 +43,7 @@ export class RenderersManager {
 
   public moveMonomer(monomer: BaseMonomer) {
     monomer.renderer?.move();
+    monomer.renderer?.drawSelection();
```

Comment:
Because without it if we have selected monomer and turn on snake mode we don't get right position of selection (green circle). It doesn't update in this case.